src/server/game/Server/Packets/LFGPackets.cpp

```diff
@@ -0,0 +1,33 @@
+/*
+ * This file is part of the TrinityCore Project. See AUTHORS file for Copyright information
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the
+ * Free Software Foundation; either version 2 of the License, or (at your
+ * option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "LFGPackets.h"
+
+void WorldPackets::LFG::LFGJoin::Read()
+{
+    _worldPacket >> Roles;
+    for (uint8 i = 0; i < 3; ++i)
```

Comment:
This is not the same code you removed from https://github.com/TrinityCore/TrinityCore/pull/25151/files#diff-ee1b5ab275f949c7fb79bc7ea2e24287L62-L64, obviously breaks LFG

Reply:
Should be ok with https://github.com/TrinityCore/TrinityCore/pull/25151/commits/0fe5155db39ecd1cd1a3b43aaf16a41a9dd56586